pkg/helper/client.go

```diff
@@ -35,6 +35,7 @@ var (
 
 const nmstateCommand = "nmstatectl"
 const vlanFilteringCommand = "vlan-filtering"
+const defaultGwRetrieveTimeout = 30
```

Comment:
I can imagine the GW reachability in the next constant may be limited by this one. Let's make all of them 120 for now and later we can pass Context around with shared timeout. What do you think?

Reply:
120 is good for me.